app.js

```diff
@@ -0,0 +1,336 @@
+//Does your browser even support local storage, man?
+function supportsLocalStorage () {
+  try {
+    return 'localStorage' in window && window['localStorage'] !== null;
+  } catch(event) {
+    return false;
+  }
+}
+//Hassle the user if their browser doesn't support local storage
+if (!supportsLocalStorage()) {
+  alert("Heeeeeeeeeeeyyyyyyyyyyyyy buddy... It would be really helpful if we could use local storage on your machine. But we can't. So, this app won't work as well for you as it will for other people. Sorry, or whatever, I guess.");
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.getElementById('registrar');
+  const input = form.querySelector('input');
+  const mainDiv = document.querySelector('.main');
+  const filterHTMLElements = {
+    ContainerDiv : filterConDiv = document.createElement('div'),
+    AttendingFilter : (
+      attendingDiv = document.createElement('div'),
+      attendingLabel = document.createElement('label'),
+      attendingCheckbox = document.createElement('input')
+    ),
+    NoShowFilter : (
+      noShowDiv = document.createElement('div'),
+      noShowLabel = document.createElement('label'),
+      noShowCheckbox = document.createElement('input')
+    ),
+    HasRespondedFilter : (
+      respondedDiv = document.createElement('div'),
+      respondedLabel = document.createElement('label'),
+      respondedCheckbox = document.createElement('input')
+    )
+  };
+  const ul = document.getElementById('invitedList');
+  let inviteeNameList = [];
```

Comment:
Okay, so potentially instead of saving the invitees as an array of strings, an array of objects might be best. You can store each person in the array along with its data like so:

```
let inviteeList = [
  {
    name: 'Kolbe',
    attending: false,
    notAttending: false,
    notes: ''
  },
  {...another person}
]
```

Reply:
Makes sense.